tap-agent/src/agent/sender_allocation.rs

```diff
@@ -599,8 +652,31 @@ impl SenderAllocationState {
             )
             .execute(&self.pgpool)
             .await
-            .map_err(|e| anyhow!("Failed to store failed receipt: {:?}", e))?;
+            .map_err(|e| anyhow!("Failed to store invalid receipt: {:?}", e))?;
         }
+        let fees = receipts
+            .iter()
+            .map(|receipt| receipt.signed_receipt().message.value)
+            .sum();
+
+        self.invalid_receipts_fees
+            .value
+            .checked_add(fees)
```

Comment:
```suggestion
        self.invalid_receipts_fees.value = self.invalid_receipts_fees
            .value
            .checked_add(fees)
```

Reply:
Thank you so much about this, I also found a bug in the tests and now they are working as expected!